alt_src/alt_src.py

```diff
@@ -757,6 +757,16 @@ def sync_repo(self):
         cmd = ['git', 'fetch', '-v', git_url, '+refs/*:refs/*']
         try:
             self.log_cmd(cmd, cwd=repo)
+        except CommandError:
+            self.logger.warning('Unable to fetch remote repo')
```

Comment:
Logging here seems to be unnecessarily verbose.
Maybe 1 warnings and 2 info are enough e.g.:
warn: Unable to fetch...
info: Removing local cache: %s
info: Re-initializing repo

Wdyt?

Reply:
so that should be enogh?
```suggestion
            except CommandError:
                    self.logger.warning('Unable to fetch remote repo')
                    self.logger.warning('Local cache exists: %s', repo)
                    # remove broken/empty dir
                    shutil.rmtree(repo)
                    # reclone repo
                    self.logger.warning('Re-initializing repo')
```

Reply:
I think it will better to log "Removing local cache: %s', repo" otherwise it's ok.